api/py/ai/chronon/repo/join_backfill.py

```diff
@@ -80,19 +74,20 @@ def run_join_part(self, join_part: str):
         export SPARK_VERSION={self.spark_version} &&
         export EXECUTOR_MEMORY={self.executor_memory} &&
         export DRIVER_MEMORY={self.driver_memory} &&
-        python3 /tmp/run.py --mode=backfill --conf=/tmp/{self.config_path} --env=production \
-        --spark-submit-path /tmp/spark_submit.sh --selected-join-parts={join_part} --ds={self.end_date}"""
+        python3 /tmp/run.py \
+--conf=/tmp/{self.config_path} --env=production --spark-submit-path /tmp/spark_submit.sh --ds={self.end_date}"""
```

Comment:
add leading whitespace here?

Reply:
synced with Pengyu offline - Line 77 and 78 is a one-line command